### ISSUE TITLE
Log pretrained model validation before fine-tuning

### ISSUE THREAD

**Issue by hbredin** [pyannote/pyannote-audio#744]
To make sure fine-tuning actually improves performance, it would be nice to run (and log) a full validation loop before training.

Related https://github.com/PyTorchLightning/pytorch-lightning/issues/1715

**Comment by dapello** [comment]
yes, it seems like in v 1.5.1, calling `trainer.validate(model)` before `trainer.fit(model)` adds the initial validation score to the logs!

**Comment by hbredin** [comment]
Not perfect, but it does the trick...

![validate](https://user-images.githubusercontent.com/1080837/154521245-fb5c1a75-9745-46e4-b1d1-718e4bb3035c.png)

cc @FrenchKrab @juanmc2005

**Comment by odedbd** [comment]
When running trainer.validate(model) before trainer.fit(model), the val dataloader is initialized twice (workers are initialized twice). In my use case, workers initialization is a costly process, so it would be beneficial if the va dataloader workers created when running trainer.validate would be reused when running trainer.fit.

Is there a way to set this up when using a DataModule as the datasource for trainer.validate and trainer.fit?

**Comment by juanmc2005** [comment]
@odedbd have you tried setting `persistent_workers=True` when instantiating the validation `DataLoader`? I think this could solve your problem.

**Comment by odedbd** [comment]
@juanmc2005 Thank you for the suggestion, I am already using persistent_workers=True and it doesn't seem to effect the creation of workers when using DataModule and running trainer.validate before trainer.fit.

I just realized I posted my comment on an issue of the pyannote-audio repository rather then that related Pytorch Lightning issue, where it probably belongs. Sorry for the confusion.
